### DISCUSSION TITLE
Invert colors shortcut doesn't work. But maybe a dark mode would be even better

### DISCUSSION THREAD

**Discussion by NicolasMaldonadoGomez** [sumatrapdfreader/sumatrapdf#2749] [Documentation]
In the documentation it says that shortcut _i_ is supposed to run CmdInvertColors, which inverts the colors of the document you are reading, but it calls the go to page menu. I haven't been able to invert colors other than manually changing the advance options file (but this is very cumbersome). I even tried to add the shortcut manually, like this:
Shortcuts [
	[
		Cmd = CmdInvertColors
		Key = w
	]
]
But that didn't work. Is there another shortcut or any other way to activate this invert colors command? Or even better, a dark mode button added to the bar, that would be great.

**Comment by GitHubRulesOK** [comment]
Inverse replacing info (now shifted i is info) was included in 3.4 and your shortcut method (works for me) should also work in more recent 3.4.#s
ensure it is the correct settings file.

The most recent cmd enhancements will generally be in 3.5 pre-release so for test/fixes use current i.e 3.5.15189

**Reply by NicolasMaldonadoGomez** [reply]
Thanks, I updated to the latest version and it worked.
